requirements.txt

```diff
@@ -4,4 +4,5 @@ nose>=1.3
 flake8>=3.5
```

Comment:
Looks like `ortools` can now be installed without any manual steps through pip. I'll have these changes in my PR onto your branch.